payments-core/src/main/java/com/stripe/android/paymentsheet/model/SupportedSavedPaymentMethod.kt

```diff
@@ -0,0 +1,71 @@
+package com.stripe.android.paymentsheet.model
+
+import android.content.res.Resources
+import androidx.annotation.DrawableRes
+import com.stripe.android.R
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.PaymentMethod
+
+/**
+ * Enum defining all types of saved payment method supported on Payment Sheet.
```

Comment:
Does this class exist separate from SupportedPaymentMethods because we don't currently support SEPA debit as a payment method to add?

Reply:
Yes, and because of the utility methods that depend on classes from `payments-core`.